### ISSUE TITLE
server/model dir should be emptied after each build (?)

### ISSUE THREAD

**Issue by manosplitsis** [georgepar/kaldi-grpc-server#5]
When building a new docker image, the model directory is copied to server/model, but the user has to remove the contents manually to build from a different model dir

**Comment by georgepar** [comment]
Yes that's by design. Imagine the case where actually the user doesn't have a backup of the model and the script goes on to delete it. Its better be annoying and make the user make the conscious decision to delete their model.

We could include a cmd line flag (-y) to bypass this, or make the script ask for permission interactively. Feel free to post a pr

**Comment by georgepar** [comment]
TODO: Add a flag to remove old models during model building
